vm-run

```diff
@@ -78,7 +79,8 @@ try:
     machine.start()
 
     if args.storage:
-        machine.add_disk(path=args.storage, type='qcow2')
+        for disk in args.storage:
```

Comment:
... you can drop the `if` here, and just leave the `for`.

Reply:
That is a lot nicer, thanks!